examples/inverse/plot_dics_source_power.py

```diff
@@ -2,7 +2,7 @@
 .. _ex-inverse-source-power:
 
 =========================================
-Compute source power using DICS beamfomer
+Compute source power using DICS beamformer
 =========================================
```

Comment:
@zuxfoucault you need to change too the length of the lines with ===

Reply:
Done! Sorry I didn't notice that.